### ISSUE TITLE
Implement smart commits

### ISSUE THREAD

**Issue by w32blaster** [w32blaster/revizor#14]
Depends on #13 

Crucible uses very interesting feature, called "Smart Commits". Docs are [here](https://confluence.atlassian.com/display/FISHEYE/Using+smart+commits#Usingsmartcommits-IntegrationwithCrucible). 

The idea is commits can contain any commands for a code review tool. Examples:
- **+review** - automatically create review for this commit
- **@username** - assign it to _username_
- **ISSUE-KEY** - associate to appropriate issue within issue tracker (separate Issue Ticket: #2 )

...and so on.

Example commit message, that could be treated as command:

```
 Fix a bug +review john@email.com,alice@email.com
```

or in multiline

```
    Fix a bug

   Withing this changeset I fixed the bug when a user logs in using a smartphone
   +review john@email.com,alice@email.com
```
## Wiki

There is ready page in our Wiki: https://github.com/w32blaster/revizor/wiki/Smart-commits
